code-security/admin_guide/scan-monitor/sbom-generation.adoc

```diff
@@ -1,83 +1,112 @@
 :toc: macro
 == Software Bill of Materials (SBOM)
 
-A Software Bill of Materials (SBOM) is a list of the components such as open source packages or IaC resources,  that make up a software or an application. The list also details the known vulnerabilities of these components and other metadata such as the license details for each package.
+A Software Bill of Materials (SBOM) is a contextualized inventory of a software or an application which lists components, libraries, and versions of all open source packages and third-party components used to build it. Insight into the component version enables you to ensure the component is updated to the latest version while parallelly ensuring the component has no vulnerability. Using the SBOM, you can also identify the software supply chain in your organization and the cascading risk effect on your infrastructure if the vulnerability is not addressed timely. Open source licensing is an additional insight SBOM generates to ensure you are legally compliant during the version updates as vulnerability remediation.
+
+Prisma Cloud scans your integrated repositories for open source packages through the package manager files found in the repository and IaC resources to generate an SBOM report. You can choose to generate the SBOM report either through a CI/CD pipeline or the console. In addition, you can also choose the SBOM generation format to be either a CSV or a CycloneDX. 
 
-On Prisma Cloud you can export open source packages found in package manager files or container images, as well as IaC resources and findings to generate an SBOM report. An SBOM report can either be an artifact from a CI/CD pipeline or from the Prisma Cloud platform for onboarded repositories.
 
 toc::[]
 
-[.task]
+=== Types of SBOM Reports
 
-=== Generate an SBOM on the Prisma Cloud Console
+Prisma Cloud currently supports the export of SBOM reports in two standardized formats, CSV and CyclondeDX.
 
-You can access the SBOM Generation wizard from either the https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-code-security/scan-monitor/supply-chain-security[Supply Chain] page or the https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-code-security/scan-monitor/development-pipelines[Development Pipelines] page.
+* CSV
++
+CSV is a comma-separated format that displays a line per vulnerability or misconfiguration. This format includes a line per resource or a package without vulnerabilities with categories across open source packages, container images, IaC components, and cloud infrastructure components.
 
+* CycloneDX output follows the scheme outlined for XML version 1.4.
++
+CycloneDX format displays vulnerabilities or misconfiguration as a scheme outlined for XML version 1.4.
```

Comment:
`CycloneDX format displays all application components including vulnerabilities and misconfigurations as a scheme outlined for https://cyclonedx.org/docs/1.4/xml/[CycloneDX XML version 1.4].`

Reply:
Updated.